src/Uno.UI/UI/Xaml/Input/PointerRoutedEventArgs.Android.cs

```diff
@@ -39,39 +51,52 @@ public PointerPoint GetCurrentPoint(UIElement relativeTo)
 		{
 			var timestamp = ToTimeStamp(_nativeEvent.EventTime);
 			var device = PointerDevice.For(Pointer.PointerDeviceType);
-			var rawPosition = new Point(_nativeEvent.RawX, _nativeEvent.RawY).PhysicalToLogicalPixels(); // Relative to the screen
-			var position = GetPosition(relativeTo);
+
+			var (rawPosition, position) = GetPositions(relativeTo);
 			var properties = GetProperties();
 
 			return new PointerPoint(FrameId, timestamp, device, Pointer.PointerId, rawPosition, position, Pointer.IsInContact, properties);
 		}
 
-		private Point GetPosition(UIElement relativeTo)
+		private (Point raw, Point relative) GetPositions(UIElement relativeTo)
 		{
-			var phyX = _nativeEvent.GetX(_nativeEvent.ActionIndex);
-			var phyY = _nativeEvent.GetY(_nativeEvent.ActionIndex);
+			var phyX = _nativeEvent.GetX(_pointerIndex);
+			var phyY = _nativeEvent.GetY(_pointerIndex);
 
+			Point raw, relative;
 			if (relativeTo == null) // Relative to the window
 			{
-				var viewCoords = new int[2];
-				_receiver.GetLocationInWindow(viewCoords);
+				var windowToReceiver = new int[2];
+				_receiver.GetLocationInWindow(windowToReceiver);
 
-				phyX += viewCoords[0];
-				phyY += viewCoords[1];
-
-				return new Point(phyX, phyY).PhysicalToLogicalPixels();
+				relative = new Point(phyX + windowToReceiver[0], phyY + windowToReceiver[1]).PhysicalToLogicalPixels();
 			}
 			else if (relativeTo == _receiver) // Fast path
 			{
-				return new Point(phyX, phyY).PhysicalToLogicalPixels();
+				relative = new Point(phyX, phyY).PhysicalToLogicalPixels();
 			}
 			else
 			{
 				var posRelToReceiver = new Point(phyX, phyX).PhysicalToLogicalPixels();
 				var posRelToTarget = UIElement.GetTransform(from: _receiver, to: relativeTo).Transform(posRelToReceiver);
 
-				return posRelToTarget;
+				relative = posRelToTarget;
 			}
+
+			// Raw coordinates are relative to the screen (easier for the gesture recognizer to track fingers for manipulations)
+			// if (ANDROID > 10)
+			// {
+			//		var raw = new Point(_nativeEvent.getRawX(_pointerIndex), _nativeEvent.getRawY(_pointerIndex)).PhysicalToLogicalPixels();
+			// }
```

Comment:
Could be `__ANDROID_10__`

Reply:
True ... it's only pseudo code for now as we don't have Android 10 support yet